### DISCUSSION TITLE
Olimex ESP32-POE-ISO-EA for base station

### DISCUSSION THREAD

**Discussion by forged3d** [ESPresense/ESPresense#815] [General]
I wondered if I would be able to use an Olimex ESP32-POE-ISO-EA with ESPresense, employing the ethernet as the communication backend instead of wi-fi to improve performance.  The external antenna variant is harder to source in the US, so before ordering and waiting for a week to get it I wanted to get feedback from the community.  Thanks in advance!

**Comment by peter-dolkens** [comment]
This works - I've got one running (non EA version) with a mmWave and a PIR sensor attached too.

**Comment by DerT94** [comment]
@peter-dolkens 
Wich "Ethernet Type" did you select?

**Reply by peter-dolkens** [reply]
Sorry, mine's on a boat back from the UK, do can't double check right now. I'll try and remember to check when it finally arrives in a month or two.

**Comment by ciscojunkiebt** [comment]
Same question here... I chose "ESP32-POE" (seemed like the most likely one to choose from) but mine continues to connect via WiFi. Not finding much ESPresense documentation on "Ethernet Type" and what they mean.

**Reply by DerT94** [reply]
"ESP32-POE" works for me.

**Reply by ciscojunkiebt** [reply]
Thanks for confirming. Think I found my problem... when I initially flashed the device I had provided the WiFi network and credentials then later tried to change it to ethernet using ESP32-POE. Seems like when I flashed a different one and did not connect to WiFi and connected directly via ethernet (using the WiFi portal from ESPresense) it seems to be working fine.

Seems mostly obvious to do it this way but it would be nice to be able to connect to both WiFi and ethernet and put a metric on the interface to prefer ethernet if available, but I understand that there are limitations to these ESP's and that might be asking too much.

Thanks for the quick response!
